Adding information about labels. (#7189)
kyma-project/test-infra: docs/how-to/how-to-add-custom-secret.md

![permissions](./secret-manager-permissions.png)

> 2. Apply labels to the secret in Secret Manager that will help us identify who is an owner of a secret. For example: `owner: teamname`

Reviewer: ```suggestion
2. Apply labels to the secret in Secret Manager that will help identify the owner of a secret. For example: `owner: team name`
```

> 3. Update External Secrets Operator yaml file.

Reviewer: ```suggestion
3. Update External Secrets Operator YAML file.
```

    Add External Secret definitions to one of the following files:
    - [external_secrets_trusted.yaml](https://github.com/kyma-project/test-infra/blob/main/prow/cluster/resources/external-secrets/external_secrets_trusted.yaml) if the secret is applied only on trusted cluster (applicable for postsubmit or release job).
    - [external_secrets_untrusted.yaml](https://github.com/kyma-project/test-infra/blob/main/prow/cluster/resources/external-secrets/external_secrets_untrusted.yaml) if the secret is applied only on untrusted cluster (applicable for presubmit job).
    - [external_secrets_workloads.yaml](https://github.com/kyma-project/test-infra/blob/main/prow/cluster/resources/external-secrets/external_secrets_workloads.yaml) if the secret is applied on both clusters (applicable for presubmit and postsubmit jobs).

> 4. Apply the Secrets manually in the Prow cluster as K8s External Secret.

Reviewer: ```suggestion
4. Apply the Secrets manually in the Prow cluster as Kubernetes External Secret.
```
